corehq/apps/domain/models.py

```diff
@@ -1157,3 +1157,11 @@ class AppReleaseModeSetting(models.Model):
     def get_settings(domain):
         domain_obj, created = AppReleaseModeSetting.objects.get_or_create(domain=domain)
         return domain_obj
+
+
+class DomainOptionToggles(models.Model):
+    """
+    Additional option toggles for a domain
+    """
+    domain = models.CharField(max_length=126, primary_key=True)
+    use_latest_build_cloudcare = models.BooleanField(default=False)
```

Comment:
I considered adding a field to HqPermissions instead, but that did not feel like a good fit for this.

Reply:
agreed

Reply:
This is made configurable at the domain level and overridable (I assume) at the user level.  I would've probably reached for application and user role (`HqPermissions`), instead.  Can you talk a little more about your rationale for these levels of config?